### ISSUE TITLE
dplyr is too old

### ISSUE THREAD

**Issue by adamhsparks** [everhartlab/SscPhenoProj#5]
Launching the binder and executing the commands in the script result in the following warning:

```r
Warning: As of rlang 0.4.0, dplyr must be at least version 0.8.0.
* dplyr 0.7.4 is too old for rlang 0.4.5.
* Please update dplyr with `install.packages("dplyr")` and restart R.
```

While that's just a warning, later, on [line 92](https://github.com/everhartlab/SscPhenoProj/blob/54c8d75ddb917866e94c06f359f3bc2b61b8de7f/Analysis.R#L92) the real errors begin.

```r
Error in mutate_impl(.data, dots) : 
  Cannot convert object to a function: [type=NULL; target=CLOSXP, SPECIALSXP, or BUILTINSXP
```

I've not investigated further.

**Comment by zkamvar** [comment]
Yeah, that is a bit irritating. I have a feeling that binder changed its spec sometime between when we released this and now because rlang 0.4.0 came out in 2019, wheras this repo was released in 2018.
